packages/sit-onyx/src/components/OnyxRadioButtonGroup/OnyxRadioButtonGroup.vue

```diff
@@ -37,7 +41,9 @@ const handleChange = (event: ChangeEvent) =>
     :disabled="props.disabled"
     @change="handleChange($event as ChangeEvent)"
   >
-    <legend v-if="props.label" class="onyx-radio-button-group__label">{{ props.label }}</legend>
+    <legend v-if="props.headline" class="onyx-radio-button-group__label">
```

Comment:
```suggestion
    <legend v-if="props.headline" class="onyx-radio-button-group__headline">
```

Would prefer to also rename the CSS class

Reply:
I don't have a preference on that^^ I guess UX would call it headline